pkg/meta/meta_test.go

```diff
@@ -931,20 +889,12 @@ func RunMetaDBTests(t *testing.T, metaDB mTypes.MetaDB, preparationFuncs ...func
 			// anonymous user
 			ctx3 := userAc.DeriveContext(context.Background())
 
-			err := metaDB.SetRepoReference(repo1, tag1, manifestDigest1, ispec.MediaTypeImageManifest)
+			err := metaDB.SetRepoReference(repo1, tag1, CreateDefaultImage().AsImageData())
 			So(err, ShouldBeNil)
 
-			err = metaDB.SetRepoReference(repo2, tag1, manifestDigest1, ispec.MediaTypeImageManifest)
+			err = metaDB.SetRepoReference(repo2, tag1, CreateDefaultImage().AsImageData())
 			So(err, ShouldBeNil)
 
-			starCount, err := metaDB.GetRepoStars(repo1)
```

Comment:
Add back the calls metaDB.GetRepoStars().
This is probably why the functions are not covered.